### DISCUSSION TITLE
[3.0.x.x] Ready for a 3.0.4.0 release

### DISCUSSION THREAD

**Discussion by mhcwebdesign** [opencart/opencart#13767] [General]
@danielkerr : Can you please tag and release the latest OC 3.0.x.x branch as version 3.0.4.0. Over the past few weeks, we have added various security patches, updates of payment methods, and other bugfixes. See also https://forum.opencart.com/viewtopic.php?p=867768#p867768

**Comment by yamlv** [comment]
can you please show what are exact security patches ?

**Comment by mhcwebdesign** [comment]
OpenCart 3.0.4.0 has now been released by Daniel. See https://github.com/opencart/opencart/compare/3.0.3.9...3.0.4.0 for a complete list of all the changes, mostly bugfixes and some security patches.
